actions/src/test/resources/application.properties

```diff
@@ -1 +1,4 @@
 test.webhookSinkUrl=""
+
+# Dummy Worker Id for tests
+event-bridge.resources.workers.id=${quarkus.uuid}
```

Comment:
I've had to "litter" the test code with a few of these since the property is needed for logging.

However I could simplify matters if I moved the configuration from `manager` (where `WorkManagerImpl` uses it) to `infra` (where the logging is based.. and is also a direct dependency of elsewhere I've had to add the property for tests including `manager`). WDYT?

Reply:
Can we avoid having the same property reset twice? It doesn't seem that different from the one we have in the manager application properties

Reply:
Done.